app/webhooks/solidus_bolt/handlers/base.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Handlers
+    class Base
```

Comment:
Thank you to have moved the Handlers under webhooks path, to reflect the RailsWay, do you mind adding `Handler` suffix to all handlers?